buildpacks/quarkus/bin/detect

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+dep_xpath='/*[local-name()="project"]'
+dep_xpath+='/*[local-name()="dependencies"]'
+dep_xpath+='/*[local-name()="dependency"]'
+dep_xpath+='/*[local-name()="artifactId" and text()="quarkus-funqy-knative-events"]'
```

Comment:
@lance note that name of the artifact changed between 1.4.0 and 1.5.0 (quarkus-funqy-knative vs. quarkus-funqy-knative-events).